src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -22,14 +22,17 @@ class Focuser(AbstractSerialFocuser):
     as they are marked with the decorator @abstractmethod, we have to override them.
     """
 
-    def __init__(self, name='Astromechanics Focuser', model='Canon EF-232', vendor_id=None,
-                 product_id=None, *args, **kwargs):
+    def __init__(self, name='Astromechanics Focuser', model='Canon EF-232', port=None,
+                 vendor_id=None, product_id=None, *args, **kwargs):
```

Comment:
This doesn't seem to work:
```
In [5]: f = Focuser(vendor_id=0x0403, product_id=0x6001)
E 06-24 04:21:49.142 NotFound: No serial ports for vendor_id=403 and product_id=6001
```

Reply:
I handled this exception but it's not clear to me it should be handled. Let me know if this still comes up on testing.

Reply:
It connected this time, but there still seems to be a problem:
```
In [4]: f = Focuser(vendor_id=0x0403, product_id=0x6001)
I 06-24 06:29:02.791 Creating PanDB panoptes
D 06-24 06:29:02.795 Focuser created: Astromechanics Focuser on /dev/ttyUSB0
D 06-24 06:29:02.801 Established serial connection to Astromechanics Focuser on /dev/ttyUSB0.
D 06-24 06:29:02.803 Initialising aperture motor
D 06-24 06:29:07.810 Aperture initialised
D 06-24 06:29:07.812 Setting focus encoder zero point
W 06-24 06:29:12.821 Astromech focuser could not get current position: ValueError("invalid literal for int() with base 10: ''")
D 06-24 06:29:17.832 Focuser has been set to encoder position 0
I 06-24 06:29:17.834 Astromechanics Focuser (XXXXXX) on /dev/ttyUSB0 initialised
D 06-24 06:29:17.841 Initializing Astromechanics Focuser

In [5]: f.position
W 06-24 06:31:32.128 Astromech focuser could not get current position: ValueError("invalid literal for int() with base 10: ''")
Out[5]: ''

In [6]: f.position
W 06-24 06:31:37.410 Astromech focuser could not get current position: error(5, 'Input/output error')
Out[6]: ''
```